src/Dash.NET.Giraffe/DashApp.fs

```diff
@@ -18,11 +18,26 @@ open Dash.NET
 open Newtonsoft.Json
 
 //Giraffe, Logging and ASP.NET specific
-type DashGiraffeConfig = {
-  HostName: string
-  LogLevel: LogLevel
-  ErrorHandler: Exception -> HttpHandler
-}
+type DashGiraffeConfig = 
+    {
+        HostName: string
+        LogLevel: LogLevel
+        ErrorHandler: Exception -> HttpHandler
+    }
+
+    static member initDefault hostname = 
```

Comment:
Makes absolutely sense to have this. Can you please adapt this PR reflecting the changes of https://github.com/plotly/Dash.NET/pull/43?